website/docs/cli/commands/vendor/vendor-pull.mdx

```diff
@@ -108,6 +108,112 @@ The following config can be used to download the `vpc` component from an AWS pub
       version: "latest"
   ```
 
+## Vendoring from SSH
+
+This is the most common way to access non-public Git repositories from automated systems because it allows access to private repositories without interactive prompts.
```

Comment:
No, this should be the least common and discouraged way of using atmos vendoring. It requires an SSH key to be set up.

Reply:
While it's easy for local development, it sucks for automation. I think we should encouraging using `GITHUB_TOKEN` instead as it works well for both.